lib/domain/bloc/controller.dart

```diff
@@ -491,7 +491,7 @@ class VideoEditorController extends ChangeNotifier {
 
     final String tempPath =
         outputDirectory ?? (await getTemporaryDirectory()).path;
-    name ??= path.basenameWithoutExtension(file.path);
+    name ??= path.basenameWithoutExtension(path.normalize(file.path));
```

Comment:
Normalize path before extracting the extension which should solve [#79](https://github.com/dart-lang/path/issues/79)